infra/discovery/src/listener/handler_identity.js

```diff
@@ -207,7 +207,8 @@ class IdentityEventHandler {
       facebookVerified: decoratedIdentity.facebookVerified || false,
       googleVerified: decoratedIdentity.googleVerified || false,
       data: { blockInfo },
-      country: decoratedIdentity.country
+      country: decoratedIdentity.country,
+      avatarUrl: decoratedIdentity.avatarUrl
```

Comment:
Is it worth just saving the IPFS hash vs the whole URL ? To save on DB space and possibly make it easier in future to change our IPFS gateway naming or what IPFS gateway to use to fetch those avatar pics ?

Reply:
oh. Nevermind ! I just realized this is an IPFS URL (e.g. ipfs://<hash>) and not an URL pointing to our IPFS gateway.